lib/common/rules.c

```diff
@@ -1487,10 +1487,21 @@ pcmk__evaluate_rules(xmlNode *xml, const pcmk_rule_input_t *rule_input,
 {
     // If there are no rules, pass by default
     int rc = pcmk_rc_ok;
+    bool have_rule = false;
 
     for (xmlNode *rule = pcmk__xe_first_child(xml, PCMK_XE_RULE, NULL, NULL);
          rule != NULL; rule = pcmk__xe_next_same(rule)) {
 
+        if (have_rule) {
+            pcmk__warn_once(pcmk__wo_multiple_rules,
```

Comment:
This is basically the same thing as `pcmk__wo_location_rules`. Feel free to keep them separate, but I think we could get away with one `wo` bit.

Reply:
nope, same reason

Reply:
well, we could use the same bit in both places, but since one was previously documented and the other not, I think separate is reasonable